pom.xml

```diff
@@ -105,6 +105,17 @@
           </descriptors>
         </configuration>
       </plugin>
+      <plugin>      
+        <groupId>org.eclipse.jetty</groupId>
+          <artifactId>jetty-maven-plugin</artifactId>
+          <version>9.4.5.v20170502</version>     
```

Comment:
Given that we're adding this today, any reason to use a 2017 build instead of the latest? 9.4.28.v20200408 appears to be latest.

Reply:
Nope, there is no special reason! This PR is just a refresh of a three years opened hanging PR 😆 . I will test the latest stable version of this plugin and update it.